Doc:Single cluster deployment application (#684)
kurator-dev/kurator: docs/content/en/docs/fleet-manager/application.md

Upon examining the respective clusters, you'll find that applications originating from the same source configuration have been distributed to different clusters based on their respective policy selector labels.

> ## Deployment Application in Single Cluster

Reviewer: ```suggestion
## Deploy Application in Host Cluster
```

> Use the following command to deploy the example application that doesn't specify an `ApplicationDestination` for Fleet. This configuration allows the application to be deployed directly on the  cluster  where kurator resides, offering a simpler deployment approach for scenarios where multi-cluster management is unnecessary.

Reviewer: ```suggestion
Use the following command to deploy the example application that doesn't specify an `ApplicationDestination`. This configuration allows the application to be deployed directly in the cluster where kurator resides, providing a simpler deployment approach for scenarios where multi-cluster management is unnecessary.
```
